### PR TITLE
embed RM summit youtube videos in docs website

### PR DESCRIPTION
**Pull request by geoknee**

### CODE REVIEW

**Comment by geoknee** [comment]
Deploy preview here https://deploy-preview-1021--nitrodocs.netlify.app/applications/0010-filecoin/#presentations

File: mkdocs.yml [hunk 1]
```diff
@@ -32,3 +32,13 @@ markdown_extensions:
           format: !!python/name:pymdownx.superfences.fence_code_format
   - pymdownx.tabbed:
       alternate_style: true
+extra:
```
Comment (NiloCK) [inline mkdocs.yml:35]: Do these changes manifest as anything visible on the deployed docs page?

**Review by NiloCK — APPROVED** [review]
:+1: 

But unclear about the changes to `mkdocs.yml`.

File: mkdocs.yml (hunk 1 above)
Comment (geoknee) [reply mkdocs.yml:35]: Yes, they add some links to the footer.

**Pull request merged**